kubernetes.go

```diff
@@ -0,0 +1,68 @@
+package kubernetes
+
+import (
+	"context"
+	"errors"
+	"path/filepath"
+
+	"github.com/loadimpact/k6/js/modules"
+	k8sTypes "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/tools/clientcmd"
+	"k8s.io/client-go/util/homedir"
+)
+
+const version = "v0.0.1"
+
+type Kubernetes struct {
+	Version     string
+	Client      *kubernetes.Clientset
+	meraOptions metav1.ListOptions
+	ctx         context.Context
+}
+
+type KubernetesOptions struct {
+	ConfigPath string
+}
+
+func (obj *Kubernetes) Init(options KubernetesOptions) error {
+	kubeconfig := options.ConfigPath
+	if kubeconfig == "" {
+		home := homedir.HomeDir()
+		if home == "" {
+			return errors.New("Home dir not found")
+		}
+		kubeconfig = filepath.Join(home, ".kube", "config")
+	}
+
+	config, err := clientcmd.BuildConfigFromFlags("", kubeconfig)
```

Comment:
I think we discussed this on the docker extension as well, but would it make sense to allow users to chose between using a kubeconfig file and passing the values needed as arguments? Some (I'd argue most) that use this as part of a CI workflow likely want to pass username, password and hostname as environment variables. They can ofc write the file in the prior step, but it will end up less clean.

Reply:
@simskij 
1) It looks we're having limitations out of k8s go client:
```
Make sure your kubectl is configured and pointed to a cluster. Run kubectl get nodes to confirm.
```
https://github.com/kubernetes/client-go/tree/master/examples/out-of-cluster-client-configuration
I can't find any options to pass auth via username/password.
2) Another type of configuration they suggest is for the case when we are inside the k8s cluster:
https://github.com/kubernetes/client-go/blob/master/examples/in-cluster-client-configuration/main.go#L42
But it's not our case, right?

Reply:
Yeah, after having a look I think you're right. Let's keep it simple for now and keep it as is. Thanks! 🙏🏼